include/runtime/instance/component.h

```diff
@@ -39,11 +41,26 @@ namespace Instance {
 using namespace AST::Component;
 
 namespace {
-void typeConvert(ValueType &VT, const ValType &ValTy) noexcept {
-  switch (ValTy.getCode()) {
+void typeConvert(ValueType &VT, const ValType &Ty) noexcept {
+  switch (Ty.getCode()) {
+  case TypeCode::I8:
+    VT.emplace<PrimValType>(PrimValType::S8);
```

Comment:
`I8` is an unsigned type. Does this mean all of the unsigned integers will be interpreted as signed integers?

Reply:
Oops, I think `I8` is signed one, the final target should be we have both at the surface.

Reply:
I though `Ix` is signed and `Ux` is unsigned.

Reply:
I see. Misunderstood.